drivers/unifiedPlatform/stworkflows/pds/workflowDataservice.go

```diff
@@ -139,6 +139,16 @@ func (wfDataService *WorkflowDataService) ValidatePdsDataServiceDeployments(depl
 	return nil
 }
 
+// GetDeploymentPodName gets the pod name for the deployment
+func (wfDataService *WorkflowDataService) GetDeploymentPodName(deploymentId string) (string, error) {
```

Comment:
This method is not requrired, line no #114 returns the deploymentPod Name and deployment